pupil-auth/config.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+require('dotenv').config()
```

Comment:
These can all go....
```
MONGO_CONNECTION_STRING
QUESTION_TIME_LIMIT
TIME_BETWEEN_QUESTIONS
Azure: {
+      Scale: process.env.SQL_AZURE_SCALE

OverridePinExpiry: process.env.hasOwnProperty('OVERRIDE_PIN_EXPIRY') ? toBool(process.env.OVERRIDE_PIN_EXPIRY) : false,
```

Reply:
OverridePinExpiry var is used in the pin-generation service for the expiry time, when creating a session for local users.

Questions:
1. QUESTION_TIME_LIMIT and TIME_BETWEEN_QUESTIONS are used in service/config.service.js as default values, but it looks like they get overwritten anyway in the config service, is that the intended way?
2. Regarding azure, do we want azure insights enabled for this microservice?

Reply:
ok, in that case leave them in.  These will eventually be removed though when we have solved the decoupling issue

Reply:
2.  yes, app insights will be required for this